### PR TITLE
Client tools cleanup

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Remove some of the client tools we don't use anymore. Leaving `rhn_check` and `rhnreg_ks` for now until we have a solution to remove `rhnreg_ks` from the `mgrbootstrap.sh` script (See https://github.com/SUSE/spacewalk/issues/18932)

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: more traditional stack tools removed
- [X] **DONE**

## Test coverage
- No tests: code removal
- [X] **DONE**

## Links

Issue(s): #
Port(s): # **add downstream PR(s), if any**

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by nodeg — APPROVED** [review]
LGTM for the QE related files.

File: python/spacewalk/server/action/Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ TOP	= ../..
 SUBDIR	= server/action
 
 SPACEWALK_FILES	= __init__ configfiles kickstart kickstart_guest packages errata hardware reboot \
-          rollback activation script rhnsd rhn_applet virt kickstart_host \
+          rollback activation script rhn_applet virt kickstart_host \
           utils scap image distupgrade clientcert channels
```
Comment (vzhestkov) [inline python/spacewalk/server/action/Makefile:9]: Looks like `activation`, `scap`, `virt` should be deleted from the list as well. Am I wrong?

**Review by vzhestkov — CHANGES_REQUESTED** [review]
I'm setting this as requested change as I think there is one point missed I put the comment inline. I could be wrong, but in this case feel free to ping me.

File: python/spacewalk/server/action/Makefile (hunk 1 above)
Comment (cbosdo) [reply python/spacewalk/server/action/Makefile:9]: right! but I'll do the reverse and do the cleanup in the server only once I have removed up2date completely.

**Review by vzhestkov — APPROVED** [review]
In general it looks good to me, the mentioned items not causing any fails while not removed from the list, it's also fine for me.

**Pull request merged**